### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by midimarcus**
Initial review of docs after template repo specs have been reviewed.
@Asi0Flammeus commit bf5b6def8293dc517ac074a14044c2caf676ada7 is duplicated and addressed in https://github.com/PlanB-Network/bitcoin-educational-content/pull/623, please don't merge it.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -19,7 +20,7 @@ The PlanB Content Repo is designed to serve as a centralized hub for high-qualit
 
 ## Supported Languages
 
-The PlanB content is available in the following languages:
+Thanks to our contributors, the PlanB content is available in many languages such as but not limited to the following ones:
```
Comment (midimarcus) [inline docs/README.md]: @Asi0Flammeus does it still make sense to list currently available languages which are visible directly on planb.network website?

File: docs/README.md [hunk 2]
```diff
@@ -39,15 +40,15 @@ As we believe in the [value-4-value model](./value-4-value-model.md), we are alw
 
 ## Repository Hierarchy
 
-The repository is organized according to the following hierarchy:
+The repository is organised according to the following hierarchy:
 
 - [Courses](./course_documentation.md): Course content are stored here and must respect some format specification 
 - [Tutorials](./tutorial_documentation.md): Tutorials are stored in categories and must respect some format specification
 - [Resources](./resource_documentation.md): Four type of resources (book, builder, conference, and podcast) are stored here and each of them has their own set of format specification 
 - [Exam](./exam_documentation.md): Exams and quizzes are stored here, and each of them has their own set of format specification
```
Comment (midimarcus) [inline docs/README.md]: @Asi0Flammeus is exam feature developed or scheduled?

File: docs/content-translation-process.md [hunk 3]
```diff
@@ -7,10 +7,10 @@ As one of the main missions of the PlanB Network Content repo is to lower down t
 If you have another approach that seems relevant, feel free to post an [issue](https://github.com/DecouvreBitcoin/sovereign-university-data/issues/new) to discuss about it.
```
Comment (midimarcus) [inline docs/content-translation-process.md]: @Asi0Flammeus how can we discuss the approach here?
Maybe we can delete these two paragraphs until we have more relevant info tu publish?

File: docs/README.md [hunk 4]
```diff
@@ -39,15 +40,15 @@ As we believe in the [value-4-value model](./value-4-value-model.md), we are alw
...
 - [Standards](./standard_documentation.md): standards are stored here and must respect some format specification
```
Comment (midimarcus) [inline docs/README.md]: @Asi0Flammeus is standards section still useful?

File: docs/README.md (hunk 1 above)
Comment (midimarcus) [reply docs/README.md]: Moreover, separate projects links are no more available on planb github (old ones are pointing to DecouvreBitcoin repo)

Comment (midimarcus) [reply docs/README.md]: Languages paragraph has been completely reviewed

**Comment by midimarcus** [comment]
@Asi0Flammeus I made a first more in-depth review of the readme and related files. Some open points for you:

- I would like to keep the how-to-translate-image.md, let me know if we have specs or tutorial I can use or I have to cover it from scratch
- content-translation-process.md: is it still useful? If so, let me know what we want to explain in the sections LLM-translator and Transleet
- course_ID_rules.md: I updated it, but I still miss CSV identifier explanation
- how-to-generate-a-bip39-id.md: it is mentioned in course_documentation.md to explain how to create chapter IDs, but it seems obsolete since chapter ID are different. How does it work now?

**Comment by midimarcus** [comment]
@Asi0Flammeus added list of disciplines in course_ID_rules.md. Please let's review it.

**Comment by Asi0Flammeus** [comment]
## Image translation

We are currently working on formalizing a workflow that it, but it is not ready yet. Therefore I think it is better to write that this is WIP, but in the meantime translation can be done manually and must respect this rule
- throughout all translations, images should have the same name and be stored in the associated subfolder (eg. `assets/fr/1.webp` and `assets/en/1.webp` are the same file but translated in two different languages), this makes the replacement in the md file easier 

## Translation Process 

This process in now entirely handle by us (PBN) and updated every week, only the proofreading in necessary (cf auto-batch-translation scripts)

## BIP39 IDs vs UUIDs

Note that we (currently) have two formats for ids
1. bip39 id which is 2 words from bip39 dictionary for identifying contributors or professors like `another-satoshi`
2. [uuids](https://www.rfc-editor.org/rfc/rfc9562.html#name-uuid-version-5) (version 4 or version 5) that is used to identified content or sub-part of the content -- already used for identifying tutorials and course's chapter and course's part

**Comment by Asi0Flammeus** [comment]
I have reviewed and updated the discipline list

**Comment by midimarcus** [comment]
@Asi0Flammeus all updated.
In case you don't have any other comment, we can mark this PR as ready for review and merge it.

**Comment by midimarcus** [comment]
@Asi0Flammeus pushed last commits about images and categories

**Pull request merged**